### ISSUE TITLE
Parquet file doesn't mark date and timestamp columns

### ISSUE THREAD

**Issue by dcory** [ClickHouse/ClickBench#7]
The Parquet file metadata doesn't include logical type information that (for instance) EventDate is a date and EventTime is a timestamp.

https://github.com/apache/parquet-format/blob/master/LogicalTypes.md

**Comment by alexey-milovidov** [comment]
Yes, these Parquet files are just converted from the source data.
It's ok to convert the types during load. Or should we add this information for convenience?

What will be the easiest way to do this?

**Comment by gortiz** [comment]
Same happen with Strings, which are just binaries that are not being marked as Strings.

Apache Pinot is quite strict about that and when importing data from parquet (which is quite faster than using tsv) the data is not imported as UTF-8 but as a byte array (because that is what parquet says)

**Comment by raunaqmorarka** [comment]
The lack of timestamp annotation https://github.com/apache/parquet-format/blob/master/LogicalTypes.md#timestamp in particular on eventtime, clienteventtime and localeventtime columns is a blocker for running this benchmark in Trino, Apache Hive and Apache Spark. All three engines reject attempts to read integers without timestamp logical annotation into timestamp column type.
Without the annotation, the engine cannot know what units the given data has been provided in, so it cannot scale or round values to account for any mismatch in the timestamp precision in the table definition vs in the file.

**Comment by alexey-milovidov** [comment]
@Iskander14yo, for now, you can use one of the following workarounds:

1. Use `clickhouse local` to convert from the source Parquet file to a file with Date and Timestamp columns.
2. Skip failing queries and submit a pull request, so we can finish it while reviewing.

**Comment by raunaqmorarka** [comment]
@alexey-milovidov why don't you consider fixing the benchmark data by regenerating the parquet files with the expected logical annotations ?
Its a straightforward one time effort which will avoid other engines from hitting this problem and having to apply various workarounds. This issue is open for more than 3 years now.

**Comment by rschu1ze** [comment]
@raunaqmorarka ClickBench is a community project, please consider converting the Parquet files by yourself and provide them for download somewhere. We'd be happy to incorporate them into the benchmark.

**Comment by rschu1ze** [comment]
While at it, we should also make sure that the updated Parquet file has an enabled Bloom filters (writing them is supported since [ClickHouse 25.2](https://clickhouse.com/blog/clickhouse-release-25-02#contributed-by-michael-kolupaev))
